runtime-management/runtime-management-impl/src/main/java/io/cloudslang/runtime/impl/python/external/ExternalPythonExecutorScheduledExecutorTimeout.java

```diff
@@ -411,6 +388,21 @@ private ProcessBuilder preparePythonProcess(TempEnvironment executionEnvironment
         return processBuilder;
     }
 
+    private ProcessBuilder preparePythonProcessForEval(String pythonPath, String evalPyCode) {
```

Comment:
As Alex says, maybe this method can be extracted in a separate utility class, and reused here and in ExternalPythonExecutorCompletableFutureTimeout class.

Reply:
To be handled by another PR, since the duplication spans across other methods as well.